docs/staking.excalidraw

```diff
@@ -0,0 +1,3261 @@
+      },
+      "seed": 1530493626,
+      "version": 973,
+      "versionNonce": 1942366202,
+      "isDeleted": false,
+      "boundElements": [
+        {
+          "type": "text",
+          "id": "0qOoFISo7yjiAbEmuaF9W"
+        }
+      ],
+      "updated": 1765285926286,
+      "link": null,
+      "locked": false
+    },
+    {
+      "id": "0qOoFISo7yjiAbEmuaF9W",
+      "type": "text",
+      "x": 310.0084561671646,
+      "y": -35.092439445573746,
+      "width": 323.93975830078125,
+      "height": 50,
+      "angle": 0,
+      "strokeColor": "#1e1e1e",
+      "backgroundColor": "transparent",
+      "fillStyle": "solid",
+      "strokeWidth": 2,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "groupIds": [],
+      "frameId": null,
+      "index": "Zz",
+      "roundness": null,
+      "seed": 1382945658,
+      "version": 779,
+      "versionNonce": 844672614,
+      "isDeleted": false,
+      "boundElements": [],
+      "updated": 1765285881521,
+      "link": null,
+      "locked": false,
+      "text": "\n    Staking by a Validator/Staker",
+      "fontSize": 20,
+      "fontFamily": 5,
+      "textAlign": "left",
+      "verticalAlign": "top",
+      "containerId": "Q3rv4sItjZTEAjLbrjc3N",
+      "originalText": "\n    Staking by a Validator/Staker",
+      "autoResize": true,
+      "lineHeight": 1.25
+    },
+    {
+      "id": "QviSBa8T85pRdTz5FZoA1",
+      "type": "rectangle",
+      "x": 303.80078125,
+      "y": -707.375,
+      "width": 1021.2929687499999,
+      "height": 571.9765625,
+      "angle": 0,
+      "strokeColor": "#1e1e1e",
+      "backgroundColor": "transparent",
+      "fillStyle": "solid",
+      "strokeWidth": 2,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "groupIds": [],
+      "frameId": null,
+      "index": "a0",
+      "roundness": {
+        "type": 3
+      },
+      "seed": 191828858,
+      "version": 504,
+      "versionNonce": 1288935674,
+      "isDeleted": false,
+      "boundElements": [
+        {
+          "type": "text",
+          "id": "HpLroQZ23uRF7UfEYxffa"
+        }
+      ],
+      "updated": 1765285705713,
+      "link": null,
+      "locked": false
+    },
+    {
+      "id": "HpLroQZ23uRF7UfEYxffa",
+      "type": "text",
+      "x": 308.80078125,
+      "y": -702.375,
+      "width": 237.17984008789062,
+      "height": 50,
+      "angle": 0,
+      "strokeColor": "#1e1e1e",
+      "backgroundColor": "transparent",
+      "fillStyle": "solid",
+      "strokeWidth": 2,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "groupIds": [],
+      "frameId": null,
+      "index": "a1",
+      "roundness": null,
+      "seed": 170902586,
+      "version": 363,
+      "versionNonce": 2051001402,
+      "isDeleted": false,
+      "boundElements": [],
+      "updated": 1765285703446,
+      "link": null,
+      "locked": false,
+      "text": "\n    Adding a validator(s)",
```

Comment:
Do I understand it correctly that we will not support this in the UI? (I would be in favor of it)